packages/utils/tsconfig.doc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {},
+  // `typescript-docs-verifier` uses `ts-node` to compile README snippets
+  // and hence it is required to disable `transpileOnly` mode as it disables
+  // all type checks. Set at root to be directly forwarded to `ts-node` until
+  // https://github.com/bbc/typescript-docs-verifier/issues/31 is resolved
+  "transpileOnly": false
```

Comment:
If you want to set this it should go under `ts-node` not on the root level. 

```jsonc
"ts-node": {
   "transpileOnly": true
}
```

Reply:
This made me think if that config is actually have any effect or not.

Reply:
this is passed directly to ts-node by the `typescript-docs-verifier` module that's why it has to be in root, see code comment and issue https://github.com/bbc/typescript-docs-verifier/issues/31